contrib/Dockerfile

```diff
@@ -40,10 +36,15 @@ RUN dpkg --add-architecture arm64 && \
 	libpulse0:arm64 \
 	libpulse0:armel \
 	libpulse0:armhf \
+	libssl-dev \
+	libssl-dev:arm64 \
+	libssl-dev:armel \
+	libssl-dev:armhf \
 	pkg-config
 
 ENV PATH="/root/.cargo/bin/:${PATH}"
-RUN curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain 1.85 -y && \
+RUN apt-get install rustup && \
```

Comment:
Does this not require a `-y` flag for non-interactive builds?
This `apt-get` command could probably be moved to the command on line 22 that already installs the rest of the packages.